src/core/frameworks/gql/queries.ts

```diff
@@ -303,3 +303,45 @@ export const GET_SPIGOT_QUERY = gql`
     }
   }
 `;
+
+// fetches all of a users positions that they borrow from
+export const GET_BORROWER_POSITIONS_QUERY = gql`
+  ${BASE_LINE_FRAGMENT}
+  ${LINE_PAGE_CREDIT_FRAGMENT}
+  ${LINE_EVENT_FRAGMENT}
+  ${BASE_SPIGOT_FRAGMENT}
+  ${SPIGOT_SUMMARY_FRAGMENT}
+  ${SPIGOT_EVENT_FRAGMENT}
+  ${ESCROW_FRAGMENT}
+
+  query getBorrowerPositions($borrower: String!) {
+    lineOfCredits(where: { borrower_contains: $borrower }) {
```

Comment:
is there a more specific/semantic query that works like `borrower_eq` instead of `borrower_contains`?

Reply:
@kibagateaux @Craigson 

You can use `borrower` like this: 
`lineOfCredits(where: { borrower: $borrower })`

I've added a commit in my PR that changes the syntax for this query as well.